### ISSUE TITLE
VVV/E Underline links

### ISSUE THREAD

**Issue by Nikomus** [neontribe/contemplating-action#191]
## Description
Using colour as the only way to determine a link is a fail of `WCAG 1.4.1 Use of colour`. We should use a conventional underline. This is something that will need to be replicated for each charity partner, but the base using this style will help for creating designs to be approved.

## Plan
- [ ] Add underlines to normal links
- [x] Create any necessary issues in other repos to amend existing designs

## Acceptance Criteria
- [ ] The default CA site has underlined text links 
- [ ] There is an acceptance criteria for underlined links in the ca-seap styling ticket
(https://github.com/neontribe/ca-seap/issues/13)
- [ ] There is an acceptance criteria for underlined links in the ca-samaritans styling ticket
(https://github.com/neontribe/ca-samaritans/issues/7)

**Comment by katjam** [comment]
closing as a won't fix. We have added underlines where indicated in the design
